uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/CRM/Tasksassignments/BAO/KeyDates.php

```diff
@@ -35,4 +24,26 @@ public function testDatesForDeletedContactsNotInResult() {
       $this->assertNotEquals($currentDate['contact_id'], $deletedContact['id']);
     }
   }
+
+  /**
+   * Creates a sample contact
+   *
+   * @param array $params
+   *
+   * @return array
+   */
+  public static function fabricateContact($params = []) {

```

Comment:
Should this function not be in a central place where other T&A tests can use it?

Reply:
Should be gone now and replaced by `ContactFabricator`